metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Study.java

```diff
@@ -209,11 +205,31 @@ public void setResources(List<WebResource> resources) {
         this.resources = resources;
     }
 
+    public void setResource(WebResource resource) {
+        List<WebResource> resources = this.getResources();
+        if (resources != null) {
```

Comment:
```suggestion - can be changed to 
        if (resources == null) {
        resources = new ArrayList<>();
        }
        resources.add(resource);
```

Reply:
Thanks, fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/f50f715fb67cb7453c0ca920d5ddfecf3a3a37da.